UnitsNet.Tests/CustomCode/TurbidityTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class TurbidityTests : TurbidityTestsBase
+    {
+        // Override properties in base class here
+        protected override double MiligramsPerLiterInOneNTU => 0.333333333;
```

Comment:
How do I verify this?

If I understand correctly, turbidity needs to refer to a specific liquid in order to convert to milligrams per liter or PPM?

https://sciencing.com/definition-automated-titration-5833327.html